File: rnd_number/Nim/rnd.nim
# Use with nim c rnd.nim and then ./rnd OR nim c -r rnd.nim to run it and build it.
# Generates and prints a random number from 1 to 1000
import random
randomize()
let rndNr = rand(1..1000)
echo "Your random number is: ", rndNr

File: rnd_number/Python3/rnd.py
# Run using python3 rnd.py
# Generates and prints a random number from 0 to 1000

import random

def main():
    rndNr = random.randint(0, 1000)
    print("Your random number is:", rndNr)

if __name__ == "__main__":
    main()
